Stack/Opc.Ua.Core/Stack/Tcp/TcpServerChannel.cs

```diff
@@ -339,6 +339,9 @@ private bool ProcessHelloMessage(ArraySegment<byte> messageChunk)
             const UInt32 kProtocolVersion = 0;
             const int kResponseBufferSize = 127;
 
+            // Communication is active on the chanell
+            LastCommTime = HiResClock.TickCount;
```

Comment:
I think it were a better design to call a method from here, e.g. UpdateLastCommTime which handles the update